samples/getting-started/teleportation/README.md

```diff
@@ -0,0 +1,61 @@
+﻿---
+page_type: sample
+languages: [qsharp, python, csharp]
+products: [qdk]
+---
+
+# Quantum Teleportation Sample #
```

Comment:
We don't need the pounds around the headings, just at the beginning. Having said that, I'm not sure if they will be rendered as # or ignored by the markdig renderer.

Reply:
I always learned the style with `#` at both ends, but I'm fine either way; happy to normalize to only leading `#` if that's more consistent.